.kitchen.yml

```diff
@@ -1,21 +1,65 @@
+<%
+rpm_based_platforms = %w(centos-5 centos-6 centos-7 fedora-25)
+deb_based_platforms = %w(ubuntu-12.04 ubuntu-14.04 ubuntu-16.04 debian-7 debian-8)
+opensuse_platforms = %w(opensuse-13.2 opensuse-leap)
+%>
+---
 driver:
   name: vagrant
+  customize:
+    cableconnected1: 'on'
 
 provisioner:
   name: chef_zero
 
+verifier:
+  name: inspec
+
 platforms:
-  - name: centos-6.8
-  - name: centos-7.2
-  - name: debian-7.11
-  - name: debian-8.6
+  - name: debian-7
+    driver:
+      box: bento/debian-7.11
```

Comment:
My intention with this change is to name the platforms in a consistent way regardless the release version we use. Using generic names as debian-7 vs debian-7.11 we can define what release of debian we want to use via the box setting and we don't need to upgrade the list of exclusions from the test suites for instance and we can also use the same platform names in dokken, AWS, etc ... this is an extra level of abstraction but is useful to decouple the test suites definitions to the execution environment based on which kitchen driver you're using.

I can change it if you don't like it, but I really think it is more maintainable.